data/admin/teledeclaration.py

```diff
@@ -96,6 +99,10 @@ class TeledeclarationAdmin(ReadOnlyAdminMixin, SimpleHistoryAdmin):
         "applicant__email",
     )
 
+    def pretty_declared_data(self, obj):
```

Comment:
nitpick : je mettrais l'adjectif à la fin

```suggestion
    def declared_data_pretty(self, obj):
```